### ISSUE TITLE
Open pop-out links

### ISSUE THREAD

**Issue by pavolzibrita** [JadenGeller/Helium#162]
Hi!

   Would it be possible that helium opens the pop-outs in new helium window? There are cases where it is impossible to past just the url directly to helium, because of session (authentification) and Helium does not show the pop-out at all. (ignores it?)

Thanks!

**Comment by ascot21** [comment]
Agreed.  This would be great for WatchESPN, FoxSportsGo and many other sports live streams.

**Comment by zeg-io** [comment]
Basically it would seem any Cable - related authentication requiring a pop-up to authenticate: http://video.foxnews.com/v/1241186546001/fox-news-channel/#sp=watch-live is a simple example...
Click DirectTV when prompted... nothing happens...screen blanks and comes back to selection every now and then.

**Comment by stek29** [comment]
Related to #9
